src/Stash/Item.php

```diff
@@ -437,7 +456,7 @@ protected function validateRecord($validation, &$record)
                 $arg2 = $argArray[2];
             }
         } else {
-            $invalidation = self::SP_NONE;
+            $invalidation = Invalidation::NONE;
```

Comment:
@tedivm AFAIR it actually looks like the default invalidation behavior did not change from NONE to PRECOMPUTE as stated in the Changelog?!